0019/README.md

```diff
@@ -152,6 +154,133 @@ Endpoint metadata get response message payload MUST be an Avro-encoded object wi
 }
 ```
 
-### Timeout and retry
+
+## Partial endpoint metadata update
+
+Completing a partial metadata update request will update or create only the endpoint metadata key-value pairs present in the request payload using [JSON Patch](https://tools.ietf.org/html/rfc6902). Metadata keys must be non-empty strings containing Latin letters, digits, and underscores only.
+
+### Partial endpoint metadata update request
+
+*Partial endpoint metadata update request* message is a [targeted message](/0003/README.md#targeted-messaging) that the client sends to the repository to patch endpoint metadata.
```

Comment:
Let's try to avoid the wide usage of the "patch" word. Instead, use the "partial update": "...that the client sends to the repository to partially update endpoint metadata."